src/settings/LocationLocations/LocationManager.js

```diff
@@ -552,13 +552,33 @@ class LocationManager extends React.Component {
       mutator,
     } = this.props;
 
-    return mutator.entries.DELETE(location).then(() => {
-      this.showCalloutMessage(location.name);
-      this.transitionToParams({
-        _path: `${match.path}`,
-        layer: null
+    mutator.holdingsEntries.reset();
+    mutator.itemEntries.reset();
+
+    const query = `permanentLocationId=${location.id} or temporaryLocationId=${location.id}`;
```

Comment:
`==` should be used since it's a strict equal and has better performance at the DB level

Reply:
yep, didn't notice this, thanks